cms/service/EvaluationService.py

```diff
@@ -1002,17 +1031,6 @@ def queue_status(self):
         return ([QueueEntry]): the list with the queued elements.
 
         """
```

Comment:
I updated the docstring. (Let me know if you wanted it to say something else.)

Reply:
Thanks, it looks almost good now. The text
>  Generally, we will see only one evaluate operation for each submission in the queue status.

still seems useful, so it could be restored (or I could do it during the merge).

Reply:
Done.